src/v3/preact.config.js

```diff
@@ -144,10 +144,13 @@ export default {
     config.resolve.alias['@okta/okta-auth-js'] = resolve(
```

Comment:
is this actually necessary? does preact not find the ESM module with its default logic?

Reply:
We've had this in for a while because Webpack 4 (Preact CLI sub-dep) doesn't deal with this properly